ng/src/web/pages/schedules/dialog.js

```diff
@@ -70,48 +63,32 @@ const TimeUnitSelect = props => {
 };
 
 const ScheduleDialog = ({
+  date,
+  duration = NO_VALUE,
+  duration_unit = 'hour',
+  hour,
+  minute,
+  period = NO_VALUE,
+  period_unit = 'hour',
+  schedule,
+  timezone = 'UTC',
+  title = _('New Schedule'),
+  visible = true,
+  onClose,
+  onSave,
+}) => {
+
+  const data = {
+    ...DEFAULTS,
```

Comment:
why is DEFAULTS still necessary? Why is name not a prop of ScheduleDialog?

Reply:
It hasn't been before, I think, and so I just adopted that. Come to think of it, it clearly can be a prop.